client/src/app/site/assignments/models/view-assignment.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseViewModel } from '../../base/base-view-model';
+import { Assignment } from '../../../shared/models/assignments/assignment';
+import { Tag } from '../../../shared/models/core/tag';
+import { User } from '../../../shared/models/users/user';
+import { Item } from '../../../shared/models/agenda/item';
+
+export class ViewAssignment extends BaseViewModel {
+    private _assignment: Assignment;
+    private _relatedUser: User[];
+    private _agendaItem: Item;
+    private _tags: Tag[];
+
+    public get assignment(): Assignment {
+        return this._assignment;
+    }
+
+    public get relatedUser(): User[] {
```

Comment:
Even if the term assignment_related_users comes from the server, we may change this to candidates. For me this wording is more precise. What do you think?

Reply:
renamed it to candidates